pages/help/[page].tsx

```diff
@@ -0,0 +1,114 @@
+import addSiteLocation from '../../content/help/addSiteLocation.md';
+import { withSites } from '~/lib/sites';
+import ReactMarkdown from 'react-markdown';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { LinkProps } from 'next/link';
+
+export const pages: any = {
+  'add-site-location': addSiteLocation,
+};
+
+type MenuLinkProps = {
+  linkProps: LinkProps;
+  label: string;
+  currentPath: string;
+};
+
+const MenuLink: React.FC<MenuLinkProps> = ({
+  linkProps,
+  label,
+  currentPath,
+}) => {
+  const textColor =
+    linkProps.href === currentPath ? 'text-amber' : 'text-white';
+  return (
+    <Link {...linkProps}>
+      <a>
+        <div
+          className={`flex transform items-center rounded-md px-4 py-2 text-gray-600 transition-colors hover:text-gray-700`}
+        >
+          <span className={`align-center mx-4 flex-1 font-medium ${textColor}`}>
+            {label}
+          </span>
+        </div>
+      </a>
+    </Link>
+  );
+};
+
+const Help = () => {
+  const router = useRouter();
+  const { page } = router.query;
+  return (
+    <div className="flex w-full">
+      <div className="flex flex-1">
```

Comment:
Could we hide this on mobile so we have more space to play with? 
```suggestion
      <div className="flex flex-1 hidden md:block">
```